internal/testutil/acc/pre_check.go

```diff
@@ -45,10 +46,26 @@ func PreCheckAtlasUsername(tb testing.TB) {
 	}
 }
 
-func PreCheckTeamsIds(tb testing.TB) {
-	if os.Getenv("MONGODB_ATLAS_TEAMS_IDS") == "" {
-		tb.Skip("`MONGODB_ATLAS_TEAMS_IDS` must be set for Projects acceptance testing")
+func PreCheckProjectTeamsIds(tb testing.TB, min int) {
+	envVar := os.Getenv("MONGODB_ATLAS_TEAMS_IDS")
+	if envVar == "" {
+		tb.Fatal("`MONGODB_ATLAS_TEAMS_IDS` must be set for Projects acceptance testing")
+		return
 	}
+	teamsIds := strings.Split(envVar, ",")
+	if count := len(teamsIds); count < min {
+		tb.Fatalf("`MONGODB_ATLAS_TEAMS_IDS` must have at least %d team ids for this acceptance testing, has %d", min, count)
+	}
+}
+
+func GetProjectTeamsIds(pos int) string {
+	envVar := os.Getenv("MONGODB_ATLAS_TEAMS_IDS")
```

Comment:
nit: can you please pass the name of the environment variable as input of the function?

Reply:
I think it's better in this way as PreCheckProjectTeamsIds and GetProjectTeamsIds works together for teamsIds and they both use the same env.var

Reply:
> work together 
I think this is an assumption that might not be always true, unless you call the `precheck...` in the `get...` method right?

Reply:
if you don't call precheck you won't check that you have the min teams you need but it will still work (returns "" when you don't have that pos).  
but if you don't call the precheck method, i don't see how passing the env.var can help there.